prestoadmin/__init__.py

```diff
@@ -17,30 +17,89 @@
 
 __version__ = '1.2'  # Make sure to update setup.py too
 
+
 import os
+import sys
+
+main_dir = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
 
 from fabric.api import env
 
+import fabric_patches
+import mode
+
+from prestoadmin.mode import get_mode, for_mode, MODE_STANDALONE, MODE_SLIDER
+from prestoadmin.util.exception import ConfigFileNotFoundError, \
+    ConfigurationError
+
+
+#
+# Subcommands common to all modes. If anybody knows why fabric_patches is in
+# the list, I'll make a note for the next person.
+#
+__all__ = ['fabric_patches', 'mode']
+
+cfg_mode = None
+try:
+    cfg_mode = get_mode()
+except ConfigFileNotFoundError as e:
+    print >>sys.stderr, \
```

Comment:
I feel like we probably don't want this to be suppressed, but honestly, I didn't even try.

It's worth noting that this gets printed in some of the unit tests too the first time the test_help modules import prestoadmin. Also, I'm highly open to suggestions for getting this to the end of the output. The UX sucks if you run pa with no args: You get this at the top, followed by a bunch of help info. The user shouldn't need to scroll up to see what they need to do to fix stuff.